influxdb/README.md

```diff
@@ -64,59 +64,397 @@ InfluxDB is a time series database built from the ground up to handle high write
+
+### Configuration
+
+InfluxDB can be configured using a mix of a config file, environment variables, and CLI options. To mount a configuration file and use it with the server, you can use this command to generate the default configuration file:
+
+```console
+$ docker run --rm influxdb influxd print-config > config.yml
+```
+
+Modify the default configuration, which will now be available under `$PWD`. Then start the InfluxDB container:
+
+```console
+$ docker run -p 8086:8086 \
+      -v $PWD/config.yml:/etc/influxdb2/config.yml:ro \
+      influxdb
+```
+
+Modify `$PWD` to be the directory where you want to store the configuration file.
+
+Individual config settings can be overridden by environment variables. The variables must be named using the format `INFLUXD_${SNAKE_CASE_NAME}`. The `SNAKE_CASE_NAME` for an option will be the option's name with all dashes (`-`) replaced by underscores (`_`), in all caps.
+
+Examples:
+
+```console
+# Config setting: bolt-path
+INFLUXD_BOLT_PATH=/root/influxdb.bolt
+# Config setting: no-tasks
+INFLUXD_NO_TASKS=true
+# Config setting: storage-wal-fsync-delay
+INFLUXD_STORAGE_WAL_FSYNC_DELAY=15m
+```
+
+Finally, all config options can be passed as CLI options:
+
+```console
+$ docker run -p 8086:8086 \
+      influxdb --storage-wal-fsync-delay=15m
+```
+
+CLI option take precedence over environment variables.
+
+Find more about configuring InfluxDB [here](https://docs.influxdata.com/influxdb/v2.0/reference/config-options/).
+
+### Database Setup
+
+InfluxDB 2.0 requires authentication. A special API exists to bootstrap the first super-user in the database, along with an initial organization and bucket. It's possible to access this API manually, or to run it automatically via environment variables.
+
+#### Manual Setup
+
+If your InfluxDB container is running locally (or on a host exposed to the network), you can perform initial setup from outside the container using either the UI or the `influx` CLI. Find more about setting up InfluxDB using these methods [here](https://docs.influxdata.com/influxdb/v2.0/get-started/#set-up-influxdb).
+
+It's also possible to perform manual setup from within the container using `docker exec`. For example, if you start the container:
+
+```console
+$ docker run -d -p 8086:8086 \
+      --name influxdb2 \
+      -v $PWD:/var/lib/influxdb2 \
+      influxdb
+```
+
+You can then run the `influx` client in the container:
+
+```console
+$ docker exec influxdb2 influx setup \
+      --username $USERNAME \
+      --password $PASSWORD \
+      --org $ORGANIZATION \
+      --bucket $BUCKET
+```
+
+Running setup from within the container will cause CLI configs to be written to `/etc/influxdb2/influx-configs`. You can use `docker cp` to copy it out of the container, and extract the token it contains. Alternatively, you could configure your initial InfluxDB run to mount `/etc/influxdb2` as a volume:
```

Comment:
did this change? it used to be just named `configs`.

Reply:
you can also grab the token with the following command and the jq utility: `$(influx auth list --json | jq -r '.[].token')`

Reply:
The Dockerfile is overriding the default directory & name for the file when it's generated. I think it's important to have it appear in a volume-path so it's not blown away on container reboot, but not as opinionated about the final filename (could revert it to `configs`, or use something new like `cli-configs`).

Reply:
Instructions added here: https://github.com/influxdata/docker-docs/pull/6/files#diff-447ed53aad4d938ca4937b1dea577b7f3d226609d99d1be13fe63e7929ce9276R115-R127